logging/google/cloud/logging/client.py

```diff
@@ -273,3 +280,40 @@ def list_metrics(self, page_size=None, page_token=None):
         """
         return self.metrics_api.list_metrics(
             self.project, page_size, page_token)
+
+    def get_default_handler(self):
+        """Return the default logging handler based on the local environment.
+
+        :rtype: :class:`logging.Handler`
+        :returns: The default log handler based on the environment
+        """
+        if os.getenv('GAE_APPENGINE_HOSTNAME') or os.getenv('GAE_INSTANCE'):
```

Comment:
1. Use `'GAE_APPENGINE_HOSTNAME' in os.environ`
1. At the very least make the env. vars. module-level constants. I would suggest putting them in `google.cloud.environment_vars`, though less churn in `core` is better